### PR TITLE
Fix behavior of CRIU_RESTORE_DISABLE_RECOVERY

### PR DESCRIPTION
**Pull request by sebratton**
On checkpoint restore failure, honor the env variable CRIU_RESTORE_DISABLE_RECOVERY.

### CODE REVIEW

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Review by helyarp — COMMENTED** [review]
Looks good.

**Comment by posmith** [comment]
L2 message review completed

**Comment by sebratton** [comment]
#build

File: dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server [hunk 1]
```diff
@@ -1395,23 +1394,9 @@ criuRestore()
 ##
 
 restoreServer()
-{
+{
```
Comment (tjwatson) [inline dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server:1397]: If you have other changes to make (like add tests), please remove this tab.

File: dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server [hunk 2]
```diff
@@ -1453,11 +1438,37 @@ restoreServer()
...
+## checkpointImageExists: return true (0) if checkpoint image exists
+##
+
+checkpointImageExists()
+{
+  if [ ! -d "${SERVER_OUTPUT_DIR}/workarea/checkpoint/image" ] ; then
+    return 1
+  fi
+
+  # check for --clean whic implies checkpoint image should be removed
```
Comment (tjwatson) [inline dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server:1459]: Fix spelling of `whic`, if you are making other changes.

**Review by tjwatson — APPROVED** [review]
Approved.  Only minor, non-functional changes suggested.  Please add test before merging.

**Comment by sebratton** [comment]
Closing as this was fixed elsewhere

**Pull request closed without merging**